examples/depthEstimation-video/sketch.js

```diff
@@ -0,0 +1,110 @@
+/*
+ * 👋 Hello! This is an ml5.js example made and shared with ❤️.
+ * Learn more about the ml5.js project: https://ml5js.org/
+ * ml5.js license and Code of Conduct: https://github.com/ml5js/ml5-next-gen/blob/main/LICENSE.md
+ *
+ * This example demonstrates basic depth estimation with ml5.depthEstimation.
+ */
+
+// Global variables
+let depthEstimator; // The ml5.js depth estimation model instance
+let video;          // The p5.js video capture element
+let depthResult;    // The object holding the latest depth estimation results
+
+// Model and runtime options
+let options = {
+    colormap: "COLOR", // Colormap for visualization ('COLOR', 'GRAYSCALE')
+    minDepth: 0,       // Minimum depth value (0-1) for normalization
+    maxDepth: 1,       // Maximum depth value (0-1) for normalization
+    // flipHorizontal: false // Set to true if using a webcam that needs flipping
+};
+
+// Video dimensions
+const videoWidth = 640;
```

Comment:
There are some examples that use `const`, but we currently are using `let` in many places where `const` would be appropriate - fyi.

Reply:
I think we should follow p5.js style which (unless it's changed?) is to use `let` everywhere. That said, I'm open minded if you think otherwise!